lalrpop/src/normalize/token_check/mod.rs

```diff
@@ -357,9 +357,7 @@ fn construct(grammar: &mut Grammar, match_block: MatchBlock) -> NormResult<()> {
             let feature = match error {
                 NamedCaptures => r#"named captures (`(?P<foo>...)`)"#,
                 NonGreedy => r#""non-greedy" repetitions (`*?` or `+?`)"#,
-                WordBoundary => r#"word boundaries (`\b` or `\B`)"#,
-                LineBoundary => r#"line boundaries (`^` or `$`)"#,
-                TextBoundary => r#"text boundaries (`^` or `$`)"#,
+                LookAround => r#"All boundaries like `\b` or `\B` or `^` or `$`"#,
```

Comment:
For consistentcy, this should start with a lower case
```suggestion
                LookAround => r#"all boundaries like `\b` or `\B` or `^` or `$`"#,
```